### DISCUSSION TITLE
How to model an air conditioner in compartment

### DISCUSSION THREAD

**Discussion by MJooLee** [firemodels/fds#12750] [Q&A]
Hello,
I would like to model a situation where an air conditioner is operating in a sealed compartment.
Initially, I thought I could set up 'supply' and 'exhaust' vents, but I understand that I need to specify a specific gas (or specify the components of air) for this. 
Instead, I want to model a situation where only the air inside the compartment is circulating when setting up the vents. 
I'm interested in how smoke behaves due to a fire in the compartment and how it interacts with the airflow from the air conditioner, as well as measuring its concentration over time. 
I'm curious how to specify the changing air inside the compartment, considering the generation of smoke over time. 
I would appreciate any advice or suggestions. 
Thank you.

**Comment by Er9y714** [comment]
I think a fan with HVAC is suitable for your case. See User guide HVAC FAN_ID and verification guide for examples.
